test/end-to-end/pages/steps.js

```diff
@@ -79,6 +79,7 @@ module.exports = () => {
 
         //Equality and diversity page
         exitEqualityAndDiversity: steps.equalityanddiversity.startpage,
+        completePCQ: steps.equalityanddiversity.pcqstartpage,
```

Comment:
What's the difference between startpage and pcqstartpage?

Reply:
We need both methods as the first one performing back button navigation and completePCQ() method performing click action of "Save and continue" button.